src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -234,7 +236,7 @@ private void initDragAndDrop() {
                 // drag'n'drop on tabs covered dnd on tabbedPane, so dnd on tabs should contain all dnds on tabbedPane
                 tabbedPane.lookupAll(".tab").forEach(tab -> {
                     tab.setOnDragOver(tabDragEvent -> {
-                        if (DragAndDropHelper.hasBibFiles(tabDragEvent.getDragboard())) {
+                        if (DragAndDropHelper.hasBibFiles(tabDragEvent.getDragboard()) || tabDragEvent.getDragboard().hasContent(DragAndDropDataFormats.GROUP)) {
```

Comment:
I would use similar methods to `DragAndDropHelper#hasBibFiles` and `DragAndDropHelper.getBibFiles` for better readability.

Reply:
These methods do not support checking having groups, so I will create some methods in the `DragAndDropHelper `class. Thanks.